vendor/photoshop/OpenColorIO_PS_Context.cpp

```diff
@@ -140,7 +152,11 @@ OpenColorIO_PS_Context::getLUTProcessor(OCIO::Interpolation interpolation, OCIO:
     
     OCIO::ConstProcessorRcPtr processor = _config->getProcessor(transform);
     
-    return processor;
+    OCIO::ConstCPUProcessorRcPtr cpu_processor = (direction == OCIO::TRANSFORM_DIR_INVERSE ?
+                                                    processor->getOptimizedCPUProcessor(OCIO::OPTIMIZATION_DEFAULT, OCIO::FINALIZATION_EXACT) :
+                                                    processor->getDefaultCPUProcessor());
+    
```

Comment:
The code above seems strange to use the Inverse or Forward direction to select the finalization flag?

Reply:
I figured for Photoshop we may as well to the exact inverse on a LUT. I guess I could use the EXACT flag on the forward transform as well, but figured it wouldn't change anything.

Reply:
Yes Brendan, the EXACT flag has no effect on forward evals (they are always exact).  If it's working fast enough for you, that's fine to use EXACT.  Give it a try on some large images to make sure.

Reply:
OK, I changed it in b6cc8072ee55915c4cba36d206e7609378cd3975 so I'm always using EXACT in Photoshop for both forward and reverse. I figure for still images Photoshop users won't mind waiting a little longer for their LUTs to run. In After Effects I leave EXACT as an optional thing.